### DISCUSSION TITLE
Align deviceclass naming to functionIDs

### DISCUSSION THREAD

**Discussion by derjoerg** [kingsleyadam/local-abbfreeathome#21] [Ideas]
Currently the switch deviceclass represents a switch_actuator in F@H so it seems that it follows the HA entity naming.

I propose to align the deviceclass naming to the functionID naming to have a better differantiation between the different channels and the possibility to react on different datapoints in F@H (which might all be the same in HA).

E.g.
| F@H naming | HA entity |
| -- | -- |
| SwitchActuator | Switch |
| SwitchSensor | Binary Sensor |
| RoomTemperatureControllerMasterWithoutFan | Climate |
| Trigger | Button |
| MovementDetector | Binary Sensor |
| BlindActuator | Cover |
...

**Comment by kingsleyadam** [comment]
You're correct, I was using the HA term for the class vs alignment to Free@Home. I'm good with this proposal and I'll convert this into an issue.
